nixos/modules/services/matrix/synapse.nix

```diff
@@ -394,6 +387,47 @@ in {
         '';
       };
 
+      log = mkOption {
+        type = types.attrsOf format.type;
+        defaultText = literalExpression defaultCommonLogConfigText;
+        description = mdDoc ''
+          Default configuration for the loggers used by `matrix-synapse` and its workers.
+          The defaults are added with the default priority which means that
+          these will be merged with additional declarations. For instance
```

Comment:
I am confused a bit because default priority would mean that I wouldn't need to use mkForce.

Reply:
No. Using option priority (`mkOption { default = defaultLogConfig; }`) means that declaring _anything_ in `services.matrix-synapse.log` _discards_ everything from `default`.

Setting everything with default priority means that you can add additional properties, but you'll need a higher priority (`mkForce`) to override existing declarations.